packages/design-system/l10n/.tx/config

```diff
@@ -1,9 +1,10 @@
 [main]
 host = https://www.transifex.com
 
-[o:owncloud-org:p:owncloud-web:r:design-system]
+[o:owncloud-org:p:owncloud-web:r:web-design-system]
```

Comment:
oh, just seeing this: you need to leave line 4 in all of the packages untouched, otherwise this will recreate all resources with new slugs on transifex. You must only add the `resource_name` property, nothing else.